staging/src/k8s.io/apiserver/pkg/server/routes/loglevel.go

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package routes
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/golang/glog"
+
+	"k8s.io/apiserver/pkg/endpoints/handlers/responsewriters"
+	"k8s.io/apiserver/pkg/server/mux"
+)
+
+// DefaultLogLevel installs the default log verbosity handler.
+type DefaultLogLevel struct {
+}
+
+type LogLevel struct {
+	Verbosity string `json:"v"`
+}
+
+// Install registers the APIServer's `/loglevel` handler.
+func (l DefaultLogLevel) Install(c *mux.PathRecorderMux) {
+	c.HandleFunc("/loglevel", func(w http.ResponseWriter, req *http.Request) {
```

Comment:
is this protected?

Reply:
Yes, 
```
func (l *Level) Set(value string) error {
	v, err := strconv.Atoi(value)
	if err != nil {
		return err
	}
	logging.mu.Lock()
	defer logging.mu.Unlock()
	logging.setVState(Level(v), logging.vmodule.filter, false)
	return nil
}
```

Reply:
I mean via authn/z. This also needs a test checking that authn/z is applied.

Reply:
Yes, it is like `/debug/pprof`, protected by authn/z . 
>This also needs a test checking that authn/z is applied.

I have to look into it, never wrote this case before.  But to make sure 
 this really acceptable.

Reply:
Done, add a case test auth

Reply:
@hzxuzhonghu Can we please query the current level using HTTP GET too?

Reply:
@dims Sorry, we cann't. because glog do not expose a method to query, if we want to do this , I think we can cache the level after first set. But we have no way to query without firstly set.

Reply:
Ack thanks @hzxuzhonghu